pkg/api/dashboard.go

```diff
@@ -272,6 +272,70 @@ func (hs *HTTPServer) getDashboardHelper(ctx context.Context, orgID int64, id in
 	return queryResult, nil
 }
 
+func (hs *HTTPServer) RestoreDashboard(c *contextmodel.ReqContext) response.Response {
+	uid := web.Params(c.Req)[":uid"]
+
+	// dash, rsp := hs.getDashboardHelper(c.Req.Context(), c.SignedInUser.GetOrgID(), 0, uid)
+	// if rsp != nil {
+	// 	return rsp
+	// }
+
+	// guardian, err := guardian.NewByDashboard(c.Req.Context(), dash, c.SignedInUser.GetOrgID(), c.SignedInUser)
+	// if err != nil {
+	// 	return response.Err(err)
+	// }
+
+	// if canRestore, err := guardian.CanDelete(); err != nil || !canRestore {
+	// 	return dashboardGuardianResponse(err)
+	// }
+
+	err := hs.DashboardService.RestoreDashboard(c.Req.Context(), uid)
+	if err != nil {
+		return response.Error(http.StatusNotFound, "Dashboard cannot be restored", err)
+	}
+
+	dash, rsp := hs.getDashboardHelper(c.Req.Context(), c.SignedInUser.GetOrgID(), 0, uid)
+	if rsp != nil {
+		return rsp
+	}
+
+	return response.JSON(http.StatusOK, util.DynMap{
+		"title":   dash.Title,
+		"message": fmt.Sprintf("Dashboard %s is restored", dash.Title),
+		"id":      dash.ID,
+		"deleted": dash.Deleted,
```

Comment:
This value should be always empty in the restore.

Reply:
Ah yes, I was going to remove that line, i was just using it to verify in postman